### DISCUSSION TITLE
Which issues should be created?

### DISCUSSION THREAD

**Discussion by An-anonymous-coder** [lichess-org/mobile#1295] [Q&A]
Since this is the rewritten version of the V1 Lichess app, it puts the issue tracker in a bit of a weird place. I have a lot of issues I *could* open, but some are features that were in the original app, others are likely planned features without their own issue trackers, etc. I don't want to spam the issue tracker with these issues. Should these issues be opened anyways? Will some issues from the V1 app be ported over automatically? Are "obvious" feature requests accepted?

**Comment by ijm8710** [comment]
I think the big think is doing a deep dive of what's been ticketed already. For example you posted something related to deep linking just now and there's already a ticket for deep linking.
Perhaps scan all the open tickets since if you filter by open only, there aren't that many.

There's probably less than 20 things that were in the old app that havent made their way into the latest beta for the new app and over half already have tickets so if you're vigilant there shouldn't be that many that don't have tickets

**Reply by An-anonymous-coder** [reply]
I didn't know it was named "deep linking", so when I had searched for duplicate issues nothing came up. Sorry about that.

**Reply by ijm8710** [reply]
Oh no worries wouldn't blame someone for a duplicate one time. I'm not a developer didn't want to mislead. Only was trying to suggest best practice since you were asking about filing a lot of issues

**Comment by veloce** [comment]
There's no need to open an issue for any existing feature not yet on the new app, either from the old mobile app or the website.

That being said, don't expect that all the features will eventually land. There is no such promise on the app. The goal is more or less to have feature parity but with mobile usage in mind. Some features don't really make sense on mobile, or should be done differently.
